File: Level 2/SACCO.py
member_name = input("Enter Member Name: ")
member_id = input("Enter Member ID: ")

contributions = []

print("Enter contributions for 6 months:")
for i in range(1, 7):
    amount = float(input(f"Month {i}: "))
    contributions.append(amount)

total_savings = sum(contributions)

print(f"\n--- SACCO Statement ---")
print(f"Member: {member_name} (ID: {member_id})")
print(f"Total Savings: KES {total_savings}")

File: Coursework/Level 2/Implementation Task/SACCO.java
import java.util.Scanner;

public class SaccoRecord {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter Member Name: ");
        String memberName = scanner.nextLine();

        System.out.print("Enter Member ID: ");
        int memberId = scanner.nextInt();

        double[] contributions = new double[6];
        double totalSavings = 0; 
        System.out.println("Enter contributions for 6 months:");
        for (int i = 0; i < 6; i++) {
            System.out.print("Month " + (i + 1) + ": ");
            contributions[i] = scanner.nextDouble();
            totalSavings += contributions[i];
        }

        System.out.println("\n--- SACCO Statement ---");
        System.out.println("Member: " + memberName + " (ID: " + memberId + ")");
        System.out.println("Total Savings: KES " + totalSavings);
        
        scanner.close();
    }
}